core/domain/stats_services.py

```diff
@@ -1117,6 +1117,54 @@ def get_learner_answer_details(entity_type, state_reference):
     return None
 
 
+def get_learner_answer_info_for_exploration(exp_id):
+    """Returns the learner answer info in dict format.
+
+    Args:
+        exp_id: str. The ID of the exploration.
+
+    Returns:
+        learner_answer_info_dict_list: list. Consist of dicts
+            with state name as key and list of leanrer answer info dicts as
+            values.
+    """
+    exploration = exp_fetchers.get_exploration_by_id(exp_id)
```

Comment:
nit: since you call the id `exp_id`, please also call this variable `exp` for consistency.

Reply:
Done